### PR TITLE
ET-3939 Creating permission set for develvopers

### PR DESCRIPTION
**Pull request by jorge-cr-13**
## What 🕵️ 🔍

- What does the pull request add and/or solve?

## References 📝 🔗

- [Jira task](https://xainag.atlassian.net/browse/)

## Reviewer checklist

*Reviewer agreement:*

* Reviewers shouldn't review a "Draft Pull Request", but rather wait for it to be marked as "Ready for Review".
* Reviewers do **not** commit or merge the pull request.
* If/when there are changes that need to be applied (applying Terraform or deploying Kubernetes manifests), the **assignee** is the one responsible for doing it.
* Use the labels `not-applied`, `partially-applied` and `fully-applied` so others know the current progress.
  * Remember to change labels accordingly once you start and finish to apply the changes.

### CODE REVIEW

File: organization/eu-central-1/buckets/permission/developer/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+dependency "xayn_yellow_bert" {
+  config_path = "../../xayn_yellow_bert"
+}
+
+
+inputs = {
+  permission_name = "s3-access"
+  actions         = ["s3:PutObject", "s3:GetObject", "s3:ListBucket"]
+  resources = [
+    "${dependency.remote_configs_repo.outputs.s3_bucket_arn}/*", "${dependency.remote_configs_repo.outputs.s3_bucket_arn}",
```
Comment (Robert-Steiner) [inline organization/eu-central-1/buckets/permission/developer/terragrunt.hcl:50]: why do we need both `<bucket_arn>` and `<bucket_arn>/*`?

File: organization/eu-central-1/buckets/permission/developer/terragrunt.hcl [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+dependency "xayn_yellow_bert" {
+  config_path = "../../xayn_yellow_bert"
+}
+
+
+inputs = {
+  permission_name = "s3-access"
+  actions         = ["s3:PutObject", "s3:GetObject", "s3:ListBucket"]
```
Comment (Robert-Steiner) [inline organization/eu-central-1/buckets/permission/developer/terragrunt.hcl:48]: we can also allow `DeleteObject`

File: organization/eu-central-1/buckets/permission/developer/terragrunt.hcl [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+dependency "xayn_static_assets" {
+  config_path = "../../xayn_static_assets"
+}
+
+dependency "xayn_yellow_bert" {
+  config_path = "../../xayn_yellow_bert"
+}
+
+
```
Comment (Robert-Steiner) [inline organization/eu-central-1/buckets/permission/developer/terragrunt.hcl:45]:
```suggestion
```

File: organization/eu-central-1/buckets/permission/developer/terragrunt.hcl (hunk 1 above)
Comment (jorge-cr-13) [reply organization/eu-central-1/buckets/permission/developer/terragrunt.hcl:50]: The `<bucket_arn>` is needed for the `ListBucket` action.

**Review by Robert-Steiner — APPROVED** [review]

**Pull request merged**